main.go

```diff
@@ -47,14 +50,25 @@ func main() {
 	}
 
 	babbageProxy := createReverseProxy(babbageURL)
+
+	dataDiscoveryURL, err := url.Parse(config.DataDiscoveryURL)
+	if err != nil {
+		log.Error(err, nil)
+		os.Exit(1)
+	}
+
+	dataDiscoveryProxy := createReverseProxy(dataDiscoveryURL)
+
 	router.HandleFunc("/", homepage.Handler(babbageProxy))
+	router.Handle("/dd{uri:(|/.*)}", dataDiscoveryProxy)
```

Comment:
The slightly complex pattern here is to allow routing both `/dd` (homepage) and `/dd/...`, but not routing unrelated pages that happen to start with dd, e.g. `/ddfoo`.